//
// Student License - for use by students to meet course requirements and
// perform academic research at degree granting institutions only.  Not
// for government, commercial, or other organizational use.
//
// acdefg_NTA_AVP.h
//
// Code generation for function 'acdefg_NTA_AVP'
//

#ifndef ACDEFG_NTA_AVP_H
#define ACDEFG_NTA_AVP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdefg_NTA_AVP(double P_init, double V_init, double A_init, double P_wayp,
                    double V_wayp, double A_wayp, double V_max, double J_max,
                    double J_min, double T, creal_T t[28]);

#endif
// End of code generation (acdefg_NTA_AVP.h)
